app/renderer/components/navbar.js

```diff
@@ -63,12 +64,27 @@ module.exports = {
             }
         },
         deleteSuite() {
-            if (this.suites.length > 1) {
-                this.suites[AppStatus.activeSuite].stopAll();
-                AppStatus.totalTasks -= this.suites[AppStatus.activeSuite].length;
-                this.suites.splice(AppStatus.activeSuite, 1);
-                this.selectTab(AppStatus.activeSuite);
-            }
+            AppAlert.toggle(
+                'Are you sure?',
+                'warning',
+                {
+                    text: 'You will not be able to recover this suite after deletion!',
+                    showCancelButton: true,
+                    confirmButtonColor: "#ee6e73",
+                    confirmButtonText: 'Yes, delete it!',
+                    cancelButtonText: 'No, keep it'
+                }
+            )
```

Comment:
No need to change this, but for next contributions, try to avoid so many almost-empty-lines:
```js
cancelButtonText: 'No, keep it'
}).then(()=>{
```
Is as readable as before, but removes 2 lines :)